components/rich_text_editor/rich_text_editor.vue

```diff
@@ -293,6 +293,10 @@ export default {
     updateEditorAttributes (attributes) {
       this.editor.setOptions({ editorProps: { attributes } });
     },
+
+    focusEditor () {
```

Comment:
@braddialpad Fixed it bit differently, tiptap had a command to focus to use, so i used ref to invoke this method directly.

The previous one had issue where the cursor was jumping from start to focus position on click.

Reply:
oh nice yeah that sounds like a better way to do it.